.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to report an error or unexpected behavior
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Open the project, run '...'
+2. Type '...' or do '...'
+3. ...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Platform:**
+ - OS: [e.g. Windows 10 x64]
+ - Version [e.g. neo-cli 2.10.2]
+
+**Additional context**
+Add any other context about the problem here.
```

Comment:
I think that there should also have a disclaimer, in which it mentions that the aforementioned information are optional.

Some people may want to just fill the issue in their manner and I believe it is still valid.

Reply:
Agreed

Reply:
Ok, has it been added or should I add?

Reply:
I just pushed, check it out. Fell free to improve.